code/modules/reagents/reagent_dispenser.dm

```diff
@@ -89,6 +91,20 @@
 		message_admins("[key_name_admin(Proj.firer)] triggered a fueltank explosion.")
 		log_game("[key_name(Proj.firer)] triggered a fueltank explosion.")
 		boom()
+		
+/obj/structure/reagent_dispensers/fueltank/attackby(obj/item/weapon/W, mob/user, params)
+	if(istype(W,/obj/item/device/assembly_holder))
+		if((W.name == "remote signaling device-igniter assembly") || (W.name == "igniter-remote signaling device assembly"))
+			if(!user.drop_item())
+				return
+			W.loc = src
+			user << "You place the igniter assembly inside the fuel tank, it's primed to blow on your signal!"
+		else
+			user << "The assembly won't fit into the tank! Only a signaller or igniter would be able to fit inside."
+	if(istype(W,/obj/item/device/assembly/igniter)
+		user << "You decide against tossing the igniter into the fuel tank, why bother if you have no way to signal it?"

```

Comment:
Ignoring the eye cancer that is this code, requiring an igniter is one thing, but restricting it to only igniter + signaller goes against the very concept of assemblies: To be modular and allow lots of options in which to trigger something. What if I wanted to use a prox sensor? Infrared beam? Timer?

Requiring the need of an igniter makes sense enough to be acceptable, but not a signaller. It should take _any_ trigger method, not just a signaller.